test/helpers.ts

```diff
@@ -216,22 +217,20 @@ export async function extractRequestFromPage(
 
       window.addEventListener(
         'resolver' as any,
-        (event: CustomEvent<string>) => {
+        (event: CustomEvent<StringifiedIsomorphicRequest>) => {
           clearTimeout(timeoutTimer)
-          resolve(JSON.parse(event.detail))
+          resolve(event.detail)
```

Comment:
We were previously parsing this, as `event.detail` is a string. Do we not need to parse it into object anymore? 🤔

Reply:
It is automatically serialized and deserialized